qemu/tests/single_driver_install.py

```diff
@@ -138,6 +138,12 @@ def run(test, params, env):
 
         session = vm.reboot(session)
 
+    if params.get("need_destroy", "no") == "yes":
```

Comment:
There is a reboot cmd in L139, after uninstall driver, in some cases, reboot is ok, in this test case we need to do destroy/create operation.
So,
if params.get("need_destroy", "no") == "yes":
xxxxx
else:
session = vm.reboot(session)

Reply:
> There is a reboot cmd in L139, after uninstall driver, in some cases, reboot is ok, in this test case we need to do destroy/create operation. So, if params.get("need_destroy", "no") == "yes": xxxxx else: session = vm.reboot(session)

@xiagao Thanks， that's reasonable, have modified it, will paste the result later.